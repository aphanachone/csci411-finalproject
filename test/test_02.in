6 0 0 0 0 5 0 11 0 0 4 0 10 12 3 8 
0 4 0 1 6 15 9 0 8 10 0 0 5 0 0 7 
13 11 0 0 3 10 8 12 9 15 14 6 0 4 2 1 
0 12 10 0 2 0 1 0 7 5 0 13 0 14 6 0 
0 0 2 0 5 6 0 7 12 0 1 16 0 8 10 0 
5 7 6 0 10 13 0 0 4 2 0 15 3 1 16 12 
10 8 13 0 16 3 12 0 14 0 7 5 2 9 0 4 
0 1 3 0 0 2 4 0 11 0 0 10 0 7 5 0 
0 15 1 0 0 0 6 5 3 8 16 12 7 0 11 0 
14 0 9 6 11 0 0 10 0 1 15 0 8 16 0 3 
11 0 0 13 12 8 0 16 6 9 5 14 0 15 4 0 
12 0 8 0 0 1 2 0 0 7 10 11 9 0 0 0 
8 3 11 0 0 12 16 0 0 14 0 7 4 0 9 0 
1 0 0 16 0 0 15 6 10 11 0 0 0 0 7 5 
9 0 4 15 0 14 5 0 0 12 2 1 11 3 0 10 
0 13 14 5 0 11 10 0 0 4 6 9 0 0 0 16 