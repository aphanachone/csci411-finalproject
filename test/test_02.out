Solution:
6 14 15 9 13 5 7 11 1 16 4 2 10 12 3 8 
2 4 16 1 6 15 9 14 8 10 12 3 5 11 13 7 
13 11 5 7 3 10 8 12 9 15 14 6 16 4 2 1 
3 12 10 8 2 16 1 4 7 5 11 13 15 14 6 9 
15 9 2 4 5 6 14 7 12 3 1 16 13 8 10 11 
5 7 6 14 10 13 11 8 4 2 9 15 3 1 16 12 
10 8 13 11 16 3 12 1 14 6 7 5 2 9 15 4 
16 1 3 12 15 2 4 9 11 13 8 10 6 7 5 14 
4 15 1 2 14 9 6 5 3 8 16 12 7 10 11 13 
14 5 9 6 11 7 13 10 2 1 15 4 8 16 12 3 
11 10 7 13 12 8 3 16 6 9 5 14 1 15 4 2 
12 16 8 3 4 1 2 15 13 7 10 11 9 5 14 6 
8 3 11 10 1 12 16 2 5 14 13 7 4 6 9 15 
1 2 12 16 9 4 15 6 10 11 3 8 14 13 7 5 
9 6 4 15 7 14 5 13 16 12 2 1 11 3 8 10 
7 13 14 5 8 11 10 3 15 4 6 9 12 2 1 16 
